README.md

```diff
@@ -4,38 +4,79 @@ Dummy Credentials repository to demonstrate how to bootstrap a tower instance wi
 
 ## Getting Started
 
-This project serves as the inventory source for the [Ansible Tower Configuration](https://github.com/integr8ly/ansible-tower-configuration) project. As this project contains sensitive information, it should only be used locally.
+This project serves as the inventory source for the [Ansible Tower Configuration](https://github.com/integr8ly/ansible-tower-configuration) project.
 
 ## Usage
 
-The example below runs the `bootstrap.yml` playbook using this project as the inventory source, where `--ask-vault-pass` is the password used to access the encrypted variables.
+The projects `SAMPLE_CREDENTIAL_CONFIG.yml` file contains a list of variables which need to be changed to suit your own environment. Once these variables have been changed, the `bootstrap.yml` playbook is used to consume this configuration file,encrypt any variables containing sensitive information using the supplied Ansible vault password, and then place each of the variables into the relevant group_vars file.
+
+## Setup
```

Comment:
Would be worth mentioning that this repository should be forked and made private before proceeding with setup steps.